scripts/netContain/scripts/contivNet.sh

```diff
@@ -2,6 +2,8 @@
 #Initialize contiv container. Start OVS and netplugin
 set -e
 
+EMBEDDED_OVS=${EMBEDDED_OVS:-"no"}
```

Comment:
I'd say remove them completely, there's no points to keep ovs running inside the contiv container, it's an upgrading/HA issue